app/migrations/0019_auto_20230523_2116.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.28 on 2023-05-24 01:16
```

Comment:
hey @aaron-lane, thanks for the effort and introducing the containerization changes :) I think the rest of the changes LGTM, just slightly worried about this migration as it contains duplicate changes to https://github.com/CodeTheChangeUBC/reBOOT/blob/master/app/migrations/0017_auto_20200903_1754.py

Could we remove change for now and check why it's introducing this first before we add it?

Reply:
I am not qualified at this point to speculate as to why this migration was automatically generated during development setup, but I noticed that it is subtly different, as it adds the `('CABLES', 'Cables')` and `('RECYCLE', 'Recycle')` tuples which are not present in the original migration.

For now, I've reverted the addition of the migration in https://github.com/CodeTheChangeUBC/reBOOT/pull/337/commits/64bdd0272ec56872d7d6738e55d1b35ca22557ad, as requested.

Reply:
Oh good catch! Feel free to squash merge when you are ready then :) we can add the migration later then